libdav1d-sys/Cargo.toml

```diff
@@ -13,3 +13,5 @@ license="BSD-2-Clause"
 
 [dependencies]
 libc = "0.2"
+[build-dependencies]
+bindgen = "0.69.4"
```

Comment:
From a quick glance at other crates it should be possible to disable default features. Maybe just `runtime` needs to be enabled - from a quick search I couldn't find what it's for.

Reply:
Without `runtime`, I get this:

```
warning: clang-sys@1.7.0: could not execute `llvm-config` one or more times, if the LLVM_CONFIG_PATH environment variable is set to a full path to valid `llvm-config` executable it will be used to try to find an instance of `libclang` on your system: "couldn't execute `llvm-config --prefix` (path=llvm-config) (error: No such file or directory (os error 2))"
```